app/src/main/java/com/example/concordia_campus_guide/Fragments/LocationFragment/LocationFragment.java

```diff
@@ -213,6 +184,23 @@ private void initFloorPlans() {
         }
     }
 
+    /**
+     * set up related to UI for the map
+     *
+     * @param mMap
+     */
+    @SuppressLint("MissingPermission")
+    private void uiSettingsForMap(GoogleMap mMap) {
+        if (ClassConstants.MY_LOCATION_PERMISSION_GRANTED) {
+            mMap.setMyLocationEnabled(true);
+        }
+        mMap.setIndoorEnabled(false);
+        mMap.getUiSettings().setMyLocationButtonEnabled(false);
+        mMap.getUiSettings().setTiltGesturesEnabled(true);
+        mMap.getUiSettings().setMapToolbarEnabled(false);
+        mMap.getUiSettings().setZoomControlsEnabled(false);
```

Comment:
This is just a suggestion but maybe we can put anything related to setting up the ui for the map somewhere else?